deployment/src/main/java/io/github/microcks/quarkus/deployment/DevServicesMicrocksProcessor.java

```diff
@@ -364,11 +364,11 @@ private RunningDevService startContainer(DevServicesConfig devServicesConfig, Do
          MicrocksContainer microcksContainer = new MicrocksContainer(dockerImageName);
 
          // Configure access to host - getting test-port from config or defaulting to 8081.
-         microcksContainer.withAccessToHost(true);
+         microcksContainer.withAccessToHost(devServiceConfig.hostAccess());
```

Comment:
`devServiceConfig` seems to not be reachable here. You probably need to add `MicrocksBuildTimeConfig` as a dependency of the build step.